prdoc/pr_6255.prdoc

```diff
@@ -0,0 +1,16 @@
+title: '[pallet-child-bounties] Index child bounties by parent bounty'
```

Comment:
prdoc needs to be updated

Reply:
You mean like this https://github.com/paritytech/polkadot-sdk/pull/6255/commits/d30c3cfc3f925ff3c1083f67a5260936669d2022 ?